tests/integration/callbacks/test_validation.py

```diff
@@ -193,3 +193,33 @@ def o2(i, s):
     dash_duo.start_server(app)
     dash_duo.wait_for_text_to_equal("#out1", "1: High")
     dash_duo.wait_for_text_to_equal("#out2", "2: High")
+
+
+def test_cbva005_tuple_args(dash_duo):
```

Comment:
TIL - I thought this was going to be broken:

https://github.com/plotly/dash/blob/fed4a9123f5d0cf6213da157bb94f5e1900c3440/dash/dependencies.py#L165

Anyway now it's tested :)

but turns out even though `a = [] + ()` is an error, `a = []; a += ()` is fine https://python-forum.io/Thread-concatenating-list-with-tuple